### ISSUE TITLE
Error when opening "asyncpixel/hypixelconstants/build/game_types.json"

### ISSUE THREAD

**Issue by hub-sdaft** [Darkflame72/asyncpixel#291]
**Describe the bug**
Whenever I instantiate `asyncpixel.Hypixel.player`, the program crashes and throws `FileNotFoundError: [Errno 2] No such file or directory: 'asyncpixel/hypixelconstants/build/game_types.json'` (on windows)

**To Reproduce**
```py
import asyncpixel
import asyncio

# Nothing changes if I remove set_event_loop_policy()
asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())

async def main():
    hypixel = asyncpixel.Hypixel(API_KEY) # API_KEY is a valid API key, I just didn't include it in the code
    player = await hypixel.player("ecbc68cfe4bf43e689aa81ab48751cc3") # That's my uuid 
    print(player)
    await hypixel.close()

asyncio.run(main())
```
This doesn't happen when calling only `hypixel.player` but anything that opens `asyncpixel/hypixelconstants/build/game_types.json`

**Current behaviour**
The program just crashes with the following traceback:
```
Traceback (most recent call last):
  File "C:\Users\sdaft\AppData\Local\Programs\Python\Python39\lib\site-packages\asyncpixel\models\player.py", line 216, in validate_game_type
    game_type = [game for game in get_game_types() if game.id == v][0]
  File "C:\Users\sdaft\AppData\Local\Programs\Python\Python39\lib\site-packages\asyncpixel\constants.py", line 61, in get_game_types        
    with open("asyncpixel/hypixelconstants/build/game_types.json") as file:
FileNotFoundError: [Errno 2] No such file or directory: 'asyncpixel/hypixelconstants/build/game_types.json'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\Users\sdaft\Documents\python\discordbot\prototype.py", line 17, in <module>
    asyncio.run(main())
  File "C:\Users\sdaft\AppData\Local\Programs\Python\Python39\lib\asyncio\runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "C:\Users\sdaft\AppData\Local\Programs\Python\Python39\lib\asyncio\base_events.py", line 642, in run_until_complete
    return future.result()
  File "c:\Users\sdaft\Documents\python\discordbot\prototype.py", line 12, in main
    await hypixel.player("ec70bcaf702f4bb8b48d276fa52a780c")
  File "C:\Users\sdaft\AppData\Local\Programs\Python\Python39\lib\site-packages\asyncpixel\hypixel.py", line 315, in player
    return Player.parse_obj(data["player"])
  File "pydantic\main.py", line 578, in pydantic.main.BaseModel.parse_obj
  File "pydantic\main.py", line 404, in pydantic.main.BaseModel.__init__
  File "pydantic\main.py", line 1040, in pydantic.main.validate_model
  File "pydantic\fields.py", line 706, in pydantic.fields.ModelField.validate
  File "pydantic\fields.py", line 913, in pydantic.fields.ModelField._apply_validators
  File "pydantic\class_validators.py", line 277, in pydantic.class_validators._generic_validator_cls.lambda3
  File "C:\Users\sdaft\AppData\Local\Programs\Python\Python39\lib\site-packages\asyncpixel\models\player.py", line 218, in validate_game_type
    game_type = [game for game in get_game_types() if game.type_name == v][0]
  File "C:\Users\sdaft\AppData\Local\Programs\Python\Python39\lib\site-packages\asyncpixel\constants.py", line 61, in get_game_types        
    with open("asyncpixel/hypixelconstants/build/game_types.json") as file:
FileNotFoundError: [Errno 2] No such file or directory: 'asyncpixel/hypixelconstants/build/game_types.json'
Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x000001A151651100>
Unclosed connector
connections: ['[(<aiohttp.client_proto.ResponseHandler object at 0x000001A1515C2EE0>, 873.453)]']
connector: <aiohttp.connector.TCPConnector object at 0x000001A151651130>
```

**Expected behaviour**
The program should just print the `player` object.

**OS:**
* OS: Windows 10 (version 20H2)
* Python version: 3.9.6 (I haven't tried with other versions, but when I do, I'll add a reply to the issue)
* Module version: 1.1.1 (Latest)

**Comment by Darkflame72** [comment]
Sorry about the delay with this, I have been on a bit of a hiatus for the last 2 months and am catching up on all the Obsidion-dev projects now. There should be a fix in the linked pull request.

**Comment by Darkflame72** [comment]
This now should be fixed as of version 1.2.0
